templates/layout-slider.php

```diff
@@ -10,7 +12,7 @@
 ]);
 ?>
 
-<div class="gallerya gallerya--slider">
+<div class="gallerya gallerya--slider" data-gallerya-navigation="<?= $show_navigation ?>">
```

Comment:
```suggestion
<div class="gallerya gallerya--slider" data-gallerya-navigation="<?= (int) $show_navigation ?>">
```
casting to an integer appears to be sensible/necessary, because:
```
$ php -r 'echo "data-foo=" . FALSE . " data-bar";'
data-foo= data-bar
```
Only a value of TRUE is casted into a string "1".

Although that might require an additional adjustment to the JS, as it seems to test for the empty string currently:
https://github.com/netzstrategen/wordpress-gallerya/blob/48be3cba572f638e7e1febf479cb1e01e19de9cd/assets/scripts/script.js#L18

A string of "0" would not be interpreted as `false` in JavaScript, unless I'm mistaken.

Reply:
Accepted your suggestion and changed the JS comparison to be `navigation == false`.

Reply:
Discussed offline with Luca. We can't do a strict comparison because when we get the value of the `data-` attribute in JavaScript, it always returns a string. We could cast it again and do a strict comparison but there's no difference to the non-strict comparison. Proceeding.